ksql-rest-app/src/test/java/io/confluent/ksql/rest/client/KsqlRestClientTest.java

```diff
@@ -200,6 +200,30 @@ public void shouldThrowOnInvalidServerAddress() {
     new KsqlRestClient("not-valid-address");
   }
 
+  @Test
+  public void shouldParseSingleServerAddress() throws Exception {
+    final String singleServerAddress = "http://singleServer:8088";
+    final URI singleServerURI = new URI (singleServerAddress);
+    try (KsqlRestClient client = new KsqlRestClient(singleServerAddress)) {
+      assertThat(client.getServerAddress(), is(singleServerURI));
+    }
+  }
+
+  @Test
+  public void shouldParseMultipleServerAddresses() throws Exception {
+    final String firstServerAddress = "http://firstServer:8088";
+    final String multipleServerAddresses = firstServerAddress + ",http://secondServer:8088";
+    final URI firstServerURI = new URI (firstServerAddress);
+    try (KsqlRestClient client = new KsqlRestClient(multipleServerAddresses)) {
+      assertThat(client.getServerAddress(), is(firstServerURI));
+    }
+  }
+
+  @Test(expected = KsqlRestClientException.class)
+  public void shouldThrowOnParsingMultipleServerAddresses() {
+    new KsqlRestClient("http://firstServer:8088,secondBuggyServer.8088");
```

Comment:
Not your fault since I assume you were simply following the pattern in `shouldThrowOnInvalidServerAddress()` above, but can we modify this test to have a more targeted check on the exception being thrown? You can do this with `ExpectedException`.

For example, if you add these lines to the top of the test file:
```
@Rule
public final ExpectedException expectedException = ExpectedException.none();
```
then your test can become:
```
@Test
public void shouldThrowIfAnyServerAddressIsInvalid() {
    expectedException.expect(KsqlRestClientException.class);
    expectedException.expectMessage("The supplied serverAddress is invalid: secondBuggyServer.8088");

    new KsqlRestClient("http://firstServer:8088,secondBuggyServer.8088");
}
```
which is a more targeted check on the error we expect.

Reply:
I've changed that as well. See the last commit. Thanks for the feedback @vcrfxia, having the opportunity to learn from you guys is unique. Very honored to learn lots of cool things \o/